modules/gha/steps.go

```diff
@@ -111,17 +111,6 @@ func (j *Job) callDaggerStep() api.JobStep {
 		// For backwards compatibility with older engines
 		env["_EXPERIMENTAL_DAGGER_CLOUD_TOKEN"] = j.PublicToken
 	}
-	for _, key := range j.envLookups() {
-		if strings.HasPrefix(key, "GITHUB_") {
-			// Inject Github context keys
-			// github.ref becomes $GITHUB_REF, etc.
-			env[key] = fmt.Sprintf("${{ github.%s }}", strings.ToLower(key))
-		} else if strings.HasPrefix(key, "RUNNER_") {
-			// Inject Runner context keys
-			// runner.ref becomes $RUNNER_REF, etc.
-			env[key] = fmt.Sprintf("${{ runner.%s }}", strings.ToLower(key))
-		}
-	}
```

Comment:
Not sure why any of this is needed - it's incorrect, since it tries to get `${{ github.github_ref }}` instead of `${{ github.ref }}`.

Also this info is all available without this block https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables. Github sets these itself without any need to do it manually.

Reply:
I suspect that the code is correct, but the comments are wrong. While GitHub does set those variables, injecting them into the container is a different story. Having those environment variables accessible in the context of the dagger release may be unnecessary, however, there are a couple examples provided to demonstrate how they could be used in different contexts:

- https://github.com/dagger/dagger/blob/8b9fbf5c643285eba1887292a79050540e399955/modules/gha/examples/go/main.go#L47
- https://github.com/dagger/dagger/blob/8b9fbf5c643285eba1887292a79050540e399955/modules/gha/examples/go/main.go#L58

Reply:
:thinking: what? These aren't actually set in any container.

This map is exclusively used as the `env` key for the call dagger step.

---

That said. Those tests are also wrong. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context

`github.repository_url` or `github.github_repository_url` isn't a real environment variable here. `github.repositoryUrl` is - but the code here won't ever generate that.

Reply:
ACK. I see what you mean.